pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -258,3 +280,14 @@ social:
 
     clearInterval(timerId);
   }
+  $(function() {
+    $('.highlight-item').mouseover(function() {
+      var caption = $(this).find('.highlight-caption');
+      caption.fadeIn();
+    });
+
+    $('.highlight-item').mouseleave(function() {
+      var caption = $(this).find('.highlight-caption');
+      caption.fadeOut();
+    });
+  });
```

Comment:
Ideally this js code would be in an included .js file rather than embedded in the haml.